src/locales/en.json

```diff
@@ -141,11 +141,11 @@
       "obtentionDate": {
         "text": "Fill in the date of each category obtained"
       },
-      "CobtentionDate": {
-        "text": "Truck - depuis le"
+      "CObtentionDate": {
+        "inputLabel": "Truck - since the"
```

Comment:
Not sure about the context but are you sure we should have `the` here?

Reply:
Also why did we remove `depuis le` in french but not in english?

Reply:
Indeed, we will just keep the name for the English version (for the moment)